LogMonitor/src/LogMonitor/ConfigFileParser.cpp

```diff
@@ -22,7 +22,7 @@
 ///
 /// \return True if the configuration file was valid. Otherwise false
 ///
-bool OpenConfigFile(_In_ const PWCHAR ConfigFileName, _Out_ LoggerSettings& Config)
+bool OpenConfigFile(_In_ const PWCHAR ConfigFileName, _In_ LoggerSettings& Config)

```

Comment:
This should remain Out - the function sets the value of settings

Reply:
Fixed this. It seems it's the reason to failing build